src/Appear.js

```diff
@@ -35,6 +35,7 @@ export default withDeck(class Appear extends React.Component {
           React.cloneElement(child, {
             key: i,
             style: {
+              ...((child.props || {}).style || {}),
```

Comment:
I think you could spread `child.props.style` after the `visibility` property to ensure that the behavior of Appear still works as expected

Reply:
If you spread the style afterwards and if in the style there is `visibility`, Appear won't do anything

Reply:
Sorry, guess my early morning brain was thinking about this the wrong way 😬